tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -282,6 +282,7 @@ class InternalCallMapHandlerTest extends TestCase
         'sqlsrv_prepare',
         'sqlsrv_query',
         'sqlsrv_server_info',
+        'ssh2_forward_accept',
```

Comment:
The extension defined 2 optional parameters (host and port) that are unused in the code and undocumented in PHP docs but are reflected.